docs/database-setup.md

```diff
@@ -1,51 +1,80 @@
-database name be the same as the username.
-
-## Testing access
-
-To test that you're able to connect (type `\q` to quit after you log in):
-
+    # test it's there
     $ psql -U <username> -h localhost <database name>
 
-You can omit the database name here if it's the same as the username.
-
-## Database URL
-
-Finally, once everything is figured out, the `DATABASE_URL` parameter you're
-prompted for during install will look something like this:
-
-    pgsql://<username>:<password>@localhost/<database name>?serverVersion=9.5
-
-The server version should be set to the version of PostgreSQL provided by your
-distro. For Ubuntu 16.04, it's '9.5'. If unsure, you can omit this parameter
-entirely and things should still work.
-
-## Other resources
-
-* <https://help.ubuntu.com/community/PostgreSQL>
-* <https://www.postgresql.org/docs/9.5/static/app-createdb.html>
-* <https://www.postgresql.org/docs/9.5/static/app-createuser.html>
+Modify your .env with following URL:
+
+    pgsql://<username>:<password>@localhost/<database name>?serverVersion=9.6
+
+Enjoy!
+
+## Real Deal
+
+Still here? Awesome. Get Some!
+
+     // Create Users
+     CREATE ROLE youwish_devuser WITH LOGIN PASSWORD 'oooh_heees_trying';
+     CREATE ROLE youwish_testuser WITH LOGIN PASSWORD 'oooh_heees_trying';
+     CREATE ROLE youwish_produser WITH LOGIN PASSWORD 'oooh_heees_trying';
+     CREATE ROLE youwish_prodadmin WITH LOGIN PASSWORD 'oooh_heees_trying';
+     // Give all other user's permissions to DB master/root user
+     GRANT youwish_devuser TO youwish_root;
+     GRANT youwish_testuser TO youwish_root;
+     GRANT youwish_produser TO youwish_root;
+     GRANT youwish_prodadmin TO youwish_root;
+     // Give admin prod role everything prod-readWrite has
+     GRANT youwish_produser TO youwish_prodadmin;
+     // Create Databases
+     CREATE DATABASE youwish_prod;
+     REVOKE ALL ON DATABASE youwish_prod FROM public;
+     CREATE DATABASE youwish_dev;
+     REVOKE ALL ON DATABASE youwish_dev FROM public;
+     CREATE DATABASE youwish_test;
+     REVOKE ALL ON DATABASE youwish_test FROM public;
+     // Prod and ProdAdmin can both connect to Prod, but Owner is ProdAdmin
+     ALTER DATABASE youwish_prod OWNER to youwish_prodadmin;
+     GRANT CONNECT ON DATABASE youwish_prod TO youwish_prodadmin;
+     GRANT CONNECT ON DATABASE youwish_prod TO youwish_produser;  
+     // Dev and Test user===admin
+     ALTER DATABASE youwish_dev OWNER to youwish_devuser;
+     ALTER DATABASE youwish_test OWNER to youwish_testuser;
+     GRANT CONNECT ON DATABASE youwish_dev TO youwish_devuser;
+     GRANT CONNECT ON DATABASE youwish_test TO youwish_testuser;
+     // For each db - create schema and default priveleges
+     \connect youwish_prod
+     CREATE SCHEMA schma AUTHORIZATION youwish_prodadmin;
+     SET search_path = schma;
+     ALTER ROLE youwish_prodadmin IN DATABASE youwish_prod SET search_path = schma;
```

Comment:
Almost certainly want the search path to include public.  So search_path = schma,public;

There are functions and other things that very well might be needed out of public.  This will make it get stuff from the intended <schema> first but then go to public if it can't find it.